rero_ils/modules/unified_entities/minters.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Persistent identifier minters."""
+from collections import namedtuple
+
+DummyMintedPID = namedtuple(
+    'DummyMintedPID',
```

Comment:
I like 'dummy' only into `tests` :-) Try to find a better name.

Reply:
I don't know how to name it ...
@rero/devteam Any suggestions are welcome

Reply:
I would use `EntityMinter`.

Reply:
thanks